src/main/java/software/amazon/cloudformation/proxy/ResourceHandlerRequest.java

```diff
@@ -46,6 +46,7 @@
     private Boolean snapshotRequested;
     private Boolean rollback;
     private Boolean driftable;
+    private Map<String, Object> features;
```

Comment:
Curious what the `Object` might look like

Reply:
ideally, any primitive/nested map